client/src/App.js

```diff
@@ -1,13 +1,24 @@
-
+import React from "react"
```

Comment:
This should probably include useState and useEffect

Reply:
Added useState and useEffect to import